test/sde/sde_dynamical.jl

```diff
@@ -19,6 +19,15 @@ g(u,p,t) = 1
     sim1  = analyticless_test_convergence(dts,prob1,BAOAB(gamma=γ),(1/2)^10;trajectories=Int(2e2),use_noise_grid=false)
     display(sim1.𝒪est)
     @test abs(sim1.𝒪est[:weak_final]-1) < 0.3
+
+    sim1  = analyticless_test_convergence(dts,prob1,ABOBA(gamma=γ),(1/2)^10;trajectories=Int(5e2),use_noise_grid=false)
```

Comment:
convergence tests are failing on CI -- did you try increasing the number of trajectories? 500 seems relatively small.

Reply:
I did increase the number of trajectories in tests and there are now passing (but longer computation time)

Reply:
Great! Maybe we should move the weak convergence tests also to the buildkite queue (@ChrisRackauckas?)

Reply:
yes for sure.